learn/user-guide/testing-ballerina-code/writing-tests.md

```diff
@@ -665,6 +665,60 @@ variables can be provided in a file named `Config.toml` located in the tests dir
 For information on using configurable variables, see
  [Configurable Variables](/learn/by-example/configurable.html).
 
+Dependencies meant to be resolved only during testing can be specified in the `Ballerina.toml` file
```

Comment:
Shall we list it under a topic `Test Dependencies`?

Reply:
Created a new Topic `Test Dependencies`